dvc/command/gc.py

```diff
@@ -55,14 +55,14 @@ def add_parser(subparsers, parent_parser):
         "--all-branches",
         action="store_true",
         default=False,
-        help="Collect garbage for all branches.",
+        help="Keep data files for the tips of all git branches.",
```

Comment:
@guysmoilov does it make sense to change the GC_HELP above as well?

Reply:
@shcheklein Good point, I tried to explain a bit more there

Reply:
Thanks @guysmoilov !